src/cloud-cli/applications/deploy-app-code.ts

```diff
@@ -92,3 +102,46 @@ export async function deployAppCode(host: string): Promise<number> {
   }
 }
 
+async function buildAppInDocker(appName: string): Promise<boolean> {
+  const logger = new GlobalLogger();
+
+  // Verify Docker is running
+  try {
+    execSync(`docker > /dev/null 2>&1`)
+  } catch (e) {
+    logger.error("Docker not found.  To deploy, please start the Docker daemon and make the `docker` command runnable without sudo.")
```

Comment:
The `Docker not found` part of the message might be misleading because an exception can be thrown for a variety of reason. Maybe we can say `Docker not available` 🤔 ?